packages/clerk-js/src/ui/localization/applyTokensToString.ts

```diff
@@ -1,8 +1,37 @@
+import { titleize } from '@clerk/shared';
+
 import { useCoreClerk, useEnvironment } from '../contexts';
-import { titleize } from '../shared';
+
+export const timeString = (val: Date | string | number, locale?: string): string => {
```

Comment:
Let's make a `localizationModifiers.ts` file, move the implementations there and export `MODIFIERS` from it.